app/views/admin/items/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1><%= @merchant.name%> Items</h1>
```

Comment:
this should probably be the admin/merchant/merchant_id/items/index.html.erb, but we couldn't figure out the nested resource